### PR TITLE
51 add docstrings

### PR DESCRIPTION
**Pull request by AmyGrimwood**
Please highlight any PEP8 issues you see

### CODE REVIEW

File: src/JSON_parsing.py [hunk 1]
```diff
@@ -1,54 +1,79 @@
...
             gene_info = input_json['genes']
             gene_list = []
             for x in gene_info:
-                #Given a key, the get() method returns paired value from dict (documentation found at https://docs.python.org/2/library/stdtypes.html in section 5.8)
+                # Given a key, the get() method returns paired value from 
+                # dict (documentation found at https://docs.python.org/2/library/stdtypes.html 
+                # in section 5.8)
                 hgnc_symbol = x.get('gene_data',{}).get('hgnc_symbol')
                 hgnc_id = x.get('gene_data',{}).get('hgnc_id')
                 GRch38_coord = x.get('gene_data',{}).get('ensembl_genes',{}).get('GRch38',{}).get('90',{}).get('location',{})
```
Comment (AbiHaddon) [inline src/JSON_parsing.py:79]: image

Getting the above pep8 issue at line 80.

File: src/JSON_parsing.py
Comment (AbiHaddon) [inline src/JSON_parsing.py:1]: Really nice use of docstring :). Good clear statements!

File: src/JSON_parsing.py (hunk 1 above)
Comment (AbiHaddon) [inline src/JSON_parsing.py:79]: line 102 characters long

File: src/JSON_parsing.py [hunk 2]
```diff
@@ -69,7 +98,6 @@ def generate_bed(self,input_json):
             location_str = "\n".join(location_list)
             bed_str = location_str.replace(':',' ').replace('-',' ')
         except KeyError:
-            print('KeyError:PanelApp output JSON doesn\' contain \'genes\' or \'location\' key')
+            print('KeyError:PanelApp output JSON doesn\'t contain \'genes\' or \'location\' key')
```
Comment (AbiHaddon) [inline src/JSON_parsing.py:101]: There are a few instances within script where line length is longer than 79 characters.

**Review by AbiHaddon — CHANGES_REQUESTED** [review]

**Comment by AbiHaddon** [comment]
@AmyGrimwood - Message from fFreddie regarding line length  'I've interpreted it as any white space before a char is counted (like "     5" is length 5) and anything after a char is not counted (like "5    " is length 1)' - so we are all on the same page

**Comment by AmyGrimwood** [comment]
> As Abi's mentioned, a few lines over 79 but docstrings look good. Need to add logging at some point as well but can leave that for another branch

Am I meant to be adding logging or does that come under Abi's logging tasks?

**Comment by AbiHaddon** [comment]
> > As Abi's mentioned, a few lines over 79 but docstrings look good. Need to add logging at some point as well but can leave that for another branch
> 
> Am I meant to be adding logging or does that come under Abi's logging tasks?

I don't we ever specified whether I would add logging to the rest of the code. Freddie has added logging to his code and I added logging to the rest of mine. I currently working on #54.

**Review by AbiHaddon — APPROVED** [review]
looks good

**Pull request merged**